soc/arm/nxp_kinetis/ke1xf/dts_fixup.h

```diff
@@ -81,19 +81,18 @@
 
 #define DT_WDT_0_NAME				DT_NXP_KINETIS_WDOG32_40052000_LABEL
 
-#define DT_SPI_1_NAME                   	DT_NXP_IMX_LPSPI_4002C000_LABEL
-#define DT_SPI_1_BASE_ADDRESS           	DT_NXP_IMX_LPSPI_4002C000_BASE_ADDRESS
-#define DT_SPI_1_IRQ                    	DT_NXP_IMX_LPSPI_4002C000_IRQ_0
-#define DT_SPI_1_IRQ_PRI                        DT_NXP_IMX_LPSPI_4002C000_IRQ_0_PRIORITY
-#define DT_SPI_1_CLOCK_NAME                     DT_NXP_IMX_LPSPI_4002C000_CLOCK_CONTROLLER
-#define DT_SPI_1_CLOCK_SUBSYS           	DT_NXP_IMX_LPSPI_4002C000_CLOCK_NAME
-
-#define DT_SPI_2_NAME                   	DT_NXP_IMX_LPSPI_4002D000_LABEL
-#define DT_SPI_2_BASE_ADDRESS           	DT_NXP_IMX_LPSPI_4002D000_BASE_ADDRESS
-#define DT_SPI_2_IRQ                    	DT_NXP_IMX_LPSPI_4002D000_IRQ_0
-#define DT_SPI_2_IRQ_PRI                        DT_NXP_IMX_LPSPI_4002D000_IRQ_0_PRIORITY
-#define DT_SPI_2_CLOCK_NAME                     DT_NXP_IMX_LPSPI_4002D000_CLOCK_CONTROLLER
-#define DT_SPI_2_CLOCK_SUBSYS           	DT_NXP_IMX_LPSPI_4002D000_CLOCK_NAME
+#define DT_SPI_0_NAME			DT_NXP_IMX_LPSPI_4002C000_LABEL
+#define DT_SPI_0_BASE_ADDRESS		DT_NXP_IMX_LPSPI_4002C000_BASE_ADDRESS
+#define DT_SPI_0_IRQ			DT_NXP_IMX_LPSPI_4002C000_IRQ_0
+#define DT_SPI_0_IRQ_PRI		DT_NXP_IMX_LPSPI_4002C000_IRQ_0_PRIORITY
+#define DT_SPI_0_CLOCK_NAME		DT_NXP_IMX_LPSPI_4002C000_CLOCK_CONTROLLER
+#define DT_SPI_0_CLOCK_SUBSYS		DT_NXP_IMX_LPSPI_4002C000_CLOCK_NAME
+#define DT_SPI_1_NAME			DT_NXP_IMX_LPSPI_4002D000_LABEL
+#define DT_SPI_1_BASE_ADDRESS		DT_NXP_IMX_LPSPI_4002D000_BASE_ADDRESS
+#define DT_SPI_1_IRQ			DT_NXP_IMX_LPSPI_4002D000_IRQ_0
+#define DT_SPI_1_IRQ_PRI		DT_NXP_IMX_LPSPI_4002D000_IRQ_0_PRIORITY
+#define DT_SPI_1_CLOCK_NAME		DT_NXP_IMX_LPSPI_4002D000_CLOCK_CONTROLLER
+#define DT_SPI_1_CLOCK_SUBSYS		DT_NXP_IMX_LPSPI_4002D000_CLOCK_NAME
```

Comment:
@henrikbrixandersen fyi

Reply:
@MaureenHelm Thanks. The current ones, introduced in ac2b8eee4ac6bd121e5446b7261bd1ac42739be2, are incorrect.